pkg/logging/logfields/logfields.go

```diff
@@ -294,7 +294,7 @@ const (
 	// Devices is the devices name
 	Devices = "devices"
 
-	//DirectRoutingDevice is the name of the direct routing device
+	// DirectRoutingDevice is the name of the direct routing device
```

Comment:
This was accidentally merged in the original PR. Can you revert it in the backport?

Reply:
Yeah, the spacing in the upstream code was fixed by my IDE. Agreed, we can let it be even though it's unrelated to the upstream PR changes.